packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1368,20 +1368,28 @@ describe('TokenListController', () => {
       );
 
       controller.startPollingByNetworkClientId('goerli');
-      jest.advanceTimersByTime(pollingIntervalTime / 2);
+      jest.advanceTimersByTime(0);
       await flushPromises();
-      expect(fetchTokenListByChainIdSpy).toHaveBeenCalledTimes(0);
-      jest.advanceTimersByTime(pollingIntervalTime / 2);
-      await flushPromises();
-
       expect(fetchTokenListByChainIdSpy).toHaveBeenCalledTimes(1);
+      await Promise.all([
+        jest.advanceTimersByTime(pollingIntervalTime / 2),
```

Comment:
It seems like you're confirming that if you advance the time but don't reach the interval, a new poll call is not run. You don't seem to have this test for other packages. Is it worth copying it?

Reply:
not sure if it's worth having them in a shared spec helper, but I've copied them to the other controllers that extend the PollingController [here](https://github.com/MetaMask/core/pull/1874/commits/d69000593048813653fcc3e9006fa08615428d16)